pkg/yurthub/util/util.go

```diff
@@ -473,3 +473,47 @@ func NewGZipReaderCloser(header http.Header, body io.ReadCloser, req *http.Reque
 		body: body,
 	}, true
 }
+
+func ParseTenantNs(certOrg string) string {
+
+	if !strings.Contains(certOrg, "openyurt:tenant:") {
+		return ""
+	}
+
+	idx := strings.LastIndex(certOrg, "openyurt:tenant:") + len("openyurt:tenant:")
```

Comment:
return strings.TrimPrefix(certOrg, "openyurt:tenant:")

Reply:
`
strings.TrimPrefix("a", "openyurt:tenant:") will return "a" as tenant ns
`

Reply:
@MrGirl we need to simplify our design, customized org should be start with `openyurt:tenant:` prefix.

Reply:
nodeclient: kubelet ask crt with system:bootstrappers 
selfnodeclient: kubelet client update crt with system:nodes 

I think it is a convention.

```
A kubelet authenticating using bootstrap tokens is authenticated as a user in the group system:bootstrappers, which is the standard method to use.
```
from [A kubelet authenticating using bootstrap tokens is authenticated as a user in the group system:bootstrappers, which is the standard method to use.](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/#initial-bootstrap-authentication)

Reply:
@MrGirl certOrg is used for tenant and comes from a parameter of yurthub, and the format is defined by `openyurt:tenant:xxx` in yurt-controller-manager. none business of bootstrap by kubelet.